src/envoy/http/authn/authn_utils.h

```diff
@@ -38,6 +38,12 @@ class AuthnUtils : public Logger::Loggable<Logger::Id::filter> {
   static bool ProcessJwtPayload(const std::string& jwt_payload_str,
                                 istio::authn::JwtPayload* payload);
 
+  // Parses the original_payload in an APToken.
```

Comment:
Is APToken a standard term or an new name defined in a design doc? Could you elaborate a little more?